Remove SnapshotTesting

stephencelis opened the pull request:
It's been causing some SPM graph issues, so let's wait for it to
stabilize a bit.

stephencelis commented on Sources/HTML/Attributes.swift:
```diff
@@ -11,6 +11,7 @@ public let id = AttributeKey<String>(key: "id")
 public let height = AttributeKey<Int>(key: "height")
 public let max = AttributeKey<Int>(key: "max")
 public let maxlength = AttributeKey<Int>(key: "maxlength")
+public let method = AttributeKey<String>(key: "min")
```
Lil typo. Also do we wanna use something like `enum Method: Value { case get, post }` for things like this?

mbrandonw replied:
LOL wow. lil typo is an UNDERSTATEMENT.

yeah we can def do that. also we have `Method` repeated at least twice so maybe can consolidate some day

stephencelis replied:
I think it's ok to be separate. `HTML.Method` only supports `GET` and `POST`.

mbrandonw replied:
oh right duh

stephencelis commented on Sources/HTML/Elements.swift:
```diff
@@ -146,6 +186,18 @@ public func p(_ content: [Node]) -> Node {
+}
+
+public func title(_ string: String) -> Node {
+  return node("title", [], [.text(string)])
```
Wondering if we should get rid of the `text` function now that we can use the enum case instead...

mbrandonw replied:
yes yes will do!

stephencelis commented on Tests/HTMLTests/FullDocumentTests.swift:
```diff
@@ -0,0 +1,124 @@
+          ]
+        ),
+        body(
+          [ HTML.`class` => "home",
```
I know we're gonna try to figure out hot to not qualify this, but the nice thing about qualifying it here is we can lose the backticks! `HTML.class` compiles just fine :D

mbrandonw replied:
ohhohohohh!

stephencelis commented on Tests/HTMLTests/HTMLTests.swift:
```diff
@@ -108,11 +101,4 @@ class HTMLTests: XCTestCase {
     XCTAssertEqual("<input disabled value=\"Hello, world!\"></input>", render(html))
   }
-
-  static var allTests = [
```
:wave:

(Would be nice to get Linux tests running in the future, though!)

mbrandonw replied:
lol man linux tests are such a pain

mbrandonw commented:
ok, I addressed all the feedback, will merge if tests pass!

stephencelis committed:
Remove SnapshotTesting

It's been causing some SPM graph issues, so let's wait for it to
stabilize a bit.

stephencelis committed:
remove other snap

mbrandonw reviewed (approved):
coo coo
